app/lib/seed_team_members.rb

```diff
@@ -0,0 +1,33 @@
+class SeedTeamMembers
+  BOARD_DATA_PATH = 'board_members.yml'.freeze
+  TEAM_DATA_PATH = 'team_members.yml'.freeze
+
+  def self.from_yaml(file_name, group)
+    members_seed_file = Rails.root.join('config', file_name)
+    members = YAML::load_file(members_seed_file)
+    members.map do |member|
+      TeamMember.find_or_create_by!(
```

Comment:
This could inadvertently create duplicate `TeamMember` records.  

The way it is written, it is trying to find a record that fits **all** of the `member` attributes, and if it does not find one, it creates a new one.

Instead we'd want it to find a `TeamMember` with the same `name`, and if it does not find one, then create it.  But if it does find one, it should **update** that record.  

The yaml file, and our codebase, does not have any duplicate names at this point, so this will work fine.  And this is a temporary solution until we have better access to prod's db.

One way to accomplish this is to pass a block to `find_or_create_by`, kind of like [this example from the docs](http://guides.rubyonrails.org/active_record_querying.html#find-or-create-by):

```ruby
Client.find_or_create_by(first_name: 'Andy') do |c|
  c.locked = false
end
```